test/manual/api/system_api_legacy_tests.sh

```
@@ -0,0 +1,531 @@
+#!/usr/bin/env bash
+# ─── Legacy Map endpoint tests ───────────────────────────────────────────────────────
+source "$(dirname "${BASH_SOURCE[0]}")/utils.sh"
+
+# Track created IDs for cleanup - use space-delimited strings to match utils.sh
+CREATED_SYSTEM_IDS=""
+CREATED_CONNECTION_IDS=""
```

Shellcheck (0.10.0)

[warning] CREATED_CONNECTION_IDS appears unused. Verify use (or export if used externally).

(SC2034)